packages/neos-ui/src/Sagas/CR/Policies/index.spec.js

```diff
@@ -0,0 +1,3 @@
+test(`write tests`, () => {
+    expect(true).toBe(true);
```

Comment:
:1st_place_medal:

Reply:
What's the point of these empty test files? We never write any tests anyways...

Reply:
:D I just followed best practice 😆